### PR TITLE
Added localization

### PR DESCRIPTION
**Pull request by humdingerb**
Still missing: generating the en.catkeys

### CODE REVIEW

**Comment by humdingerb** [comment]
I don't know how to generate the en.catkeys with the cmake buildsystem. Or how to add the translated catkeys, come to think of it...

File: Source/RunnerAddOn.cpp [hunk 1]
```diff
@@ -237,13 +246,13 @@ populate_menu(BMessage* message, BMenu* menu, BHandler* handler)
 	BMenu* prefsSubMenu = NULL;
 	builder
 		.AddSeparator()
-		.AddMenu("Preferences & Help")
+		.AddMenu(B_TRANSLATE("Settings & Help"))
```
Comment (augiedoggie) [inline Source/RunnerAddOn.cpp:249]: I'm still debating this one. There are still quite a few apps that refer to it as "Preferences".  Tracker & Deskbar for example.

**Comment by augiedoggie** [comment]
> I don't know how to generate the en.catkeys with the cmake buildsystem. Or how to add the translated catkeys, come to think of it...

It has to be enabled in the cmake files.  You can see an example in the [Beezer build](https://github.com/Teknomancer/beezer/blob/master/Source/Beezer/CMakeLists.txt#L79) files.  Then the build could be configured with `cmake -DHAIKU_ENABLE_I18N=ON` or that variable could be permanently set to ON in the cmake files.

I can take care of it later.  That part of the build system doesn't get much use so there may be some other steps required.

File: Source/RunnerAddOn.cpp (hunk 1 above)
Comment (humdingerb) [reply Source/RunnerAddOn.cpp:249]: Right, sorry I missed pointing out this change.

IMO, and I think at least Haiku apps generally adhere to that, "Preferences" are the applets found in Deskbar's  "Preferences" menu, or are system component preferences like for Tracker and Deskbar.
"Application-internal" preferences are "Settings". 

Things become difficult when it comes to "Options". :)
Those are best avoided, but maybe those would be options that refer to the current document or view, and are separate from the "app-internal" Settings. At least that's my current thinking...

**Comment by humdingerb** [comment]
Appreciated. 👍

**Pull request merged**